queries.yaml

```diff
@@ -516,32 +516,24 @@ post_process:
         - shield_text
         - all_networks
         - all_shield_texts
-        - bicycle_network
-        - bicycle_shield_text
-        - all_bicycle_networks
-        - all_bicycle_shield_texts
-        - all_walking_networks
-        - all_walking_shield_texts
         - bus_network
         - bus_shield_text
-        - all_bus_networks
-        - all_bus_shield_texts
       where: >-
         (kind == 'rail' and zoom < 15) or
         (kind == 'minor_road' and zoom < 14) or
         (kind == 'major_road' and zoom <  7)
-  # previous tests assume we keep walking network information until zoom 11
   - fn: vectordatasource.transform.drop_properties
     params:
       source_layer: roads
       start_zoom: 0
-      end_zoom: 10
+      end_zoom: 13
       properties:
-        - walking_network
-        - walking_shield_text
```

Comment:
Fixed in 7cd5ffcbfc7eab939a55d22f35596285ee1bee54. 

I also removed the path name at early zooms to enable more line merging – that was necessary for v1.0 when we didn't have the walking network information but is now getting in the way.